test/import/import-crd.test.ts

```diff
@@ -465,3 +467,52 @@ test('given a prefix, we can import two crds with the same group id', async () =
 
 
 });
+
+describe('cdk8s.yaml file', () => {
+  const jenkinsCRD: ImportSpec = {
+    source: 'https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/crds/jenkins.io_jenkins_crd.yaml',
+  };
+
+  const importOptions: ImportOptions = {
+    targetLanguage: Language.TYPESCRIPT,
+    outdir: path.join(__dirname, 'crd-imports'),
```

Comment:
Never use `__dirname` is a working directory in tests - it prevents them from running in parallel (imagine two tests, one deleting this dir and one reading from it...). Instead, each test should create a temporary working directory via `fs.mkdtempSync`